plugins/modules/provision_workflow_manager.py

```diff
@@ -96,6 +96,42 @@
                 interface_name:
                     description: Ip Address allocated to the interface
                     type: str
+        interfaces:
```

Comment:
What is the difference between interfaces and dynamic_interfaces?

Reply:
Madhan…The dynamic interface is for wireless api which uses version 2.3.5.3. This interfaces is for new api of wireless which uses version 2.3.7.6